x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_permissions.ts

```diff
@@ -214,3 +225,22 @@ async function universalProfilingPermissions(packagePolicyId: string): Promise<[
     },
   ];
 }
+
+function apmPermissions(packagePolicyId: string): [string, SecurityRoleDescriptor] {
+  return [
+    packagePolicyId,
+    {
+      cluster: ['cluster:monitor/main'],
+      indices: [
+        {
+          names: ['traces-*', 'logs-*', 'metrics-*'],
```

Comment:
Can't we just have `traces-apm*`, `logs-apm*`, `metrics-apm*`? Also

Reply:
Nope, since 8.10 apm-server has broader privileges to enable data stream routing: https://github.com/elastic/apm-server/pull/11168